src/main/java/org/zalando/nakadi/service/subscription/SubscriptionService.java

```diff
@@ -104,7 +108,18 @@ public SubscriptionService(final SubscriptionDbRepository subscriptionRepository
                 .filter(Optional::isPresent)
                 .map(Optional::get)
                 .forEach(eventType -> client.checkScopes(eventType.getReadScopes()));
-
+        final int totalPartitions = eventTypeMapping.values().stream()
+                .filter(Optional::isPresent)
+                .map(Optional::get)
+                .map(et -> topicRepository.listPartitionNames(et.getTopic()))
+                .mapToInt(List::size)
+                .sum();
+        if (totalPartitions > maxSubscriptionPartitions) {
+            final String message = String.format(
+                    "total partition count for subscription is %d while only %d allowed",
```

Comment:
I would suggest `total partition count for subscription is %d, but the maximum partition count is %d`